website/content/en/docs/helm/reference/advanced_features.md

```diff
@@ -110,3 +119,9 @@ spec:
   service:
     port: 8080
 ```
+
+Note that, after we set this annotation as `True` when an spec be updated, for example `spec.replicaCount:3`, this CR will be reconciled and then, we will be able to check that was used to `force` the release upgrade. See: 
```

Comment:
```suggestion
Setting this annotation to `True` and updating the spec (for example changing to `spec.replicaCount:3`) will cause this CR to be reconciled and upgraded with the `force` option. This can be verified in the log message when an upgrade succeeds: 
```